recipes/urllib3/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.16" %}
+
+package:
+  name: urllib3
+  version: {{ version }}
+source:
+  fn: urllib3-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/u/urllib3/urllib3-{{ version }}.tar.gz
+  md5: fcaab1c5385c57deeb7053d3d7d81d59
```

Comment:
> You should be upgrading to SHA2 or Blake2 sometime soon.

:+1: 

We recommend `sha256` BTW. The `mdf5` is the "lazy choice" b/c it comes from PyPI and does not require a manual edit from a trusted source.